src/monthlychallenges/aug-2021/index.njk

```diff
@@ -0,0 +1,79 @@
+--- 
+title: 'Monthly Theme & Challenge for August, 2021: Healthy Habits for Healthy Devs'
+description: 'August challenge -> This month challenge is all about nourish our body, mind and spirit and give a step forward to become a healthier dev'
+layout: layouts/simple.njk
+date: 2021-08-01
```

Comment:
small grammer error:

This month's challenge is all about nourishing our bodies, minds, and spirits so that we can become healthier developers.